totest-manager.py

```diff
@@ -446,13 +525,19 @@ def _release(self, set_release=None):
         for cd in self.main_products:
             self._release_package(self.project, cd, set_release=set_release)
 
+        if self.docker_packages:
+            repo_url = self._get_repo_mirror(snapshot) + "/suse"
```

Comment:
This will break as soon as we switch to the new product builder

Reply:
I know - I'll probably introduce autodetection for that instead.